index.html

```diff
@@ -97,5 +98,27 @@ <h2 id="poll" class="question">Qual ficou mais animal?</h2>
 
       <button class="square">Votar</button>
     </div>
+    <hr>
+    <div class="about" id="about">
+      <h2 id="about-us">Quem Somos</h2>
+        <p>
+          Somos três amigos com grandes sonhos. Três desenvolvedores que acreditam na tecnologia como ferramenta de transformação e revolução social. E dentre vários sonhos e projetos, existia um desejo muito grande de ajudar os animais de rua, decidimos então tirar esse lindo projeto do papel.
+        </p>
+      <h2 id="mission">Nossa Missão</h2>
+        <p>
+          A missão do PeloPêlo é criar um impacto significativo na vida dos animais por meio da tecnologia e inovação. Acreditamos que todos os seres vivos merecem cuidado, compaixão e uma chance de viver uma vida feliz e saudável. Trabalhamos para apoiar as organizações de proteção animal e contribuir para um mundo onde todos os animais sejam tratados com respeito e dignidade.
+        </p>
+      <h2 id="values">Valores</h2>
+        <h3 class="compassionValue">Compaixão:</h3>
```

Comment:
unnecessary classes, when needed, let's use the convention `compassion-value` instead `compassionValue`